Utilities/bootstrap

```diff
@@ -476,10 +476,10 @@ def create_bootstrap_files(sandbox_path, args):
         elif target.is_swift and not target.is_library:
             link_output_path = os.path.join(bin_dir, target.name)
 
-            link_command = [args.swiftc_path,
+            link_command = [pipes.quote(args.swiftc_path),
                             '-o', pipes.quote(link_output_path)]
             if args.sysroot:
-                link_command.extend(["-sdk", args.sysroot])
+                link_command.extend(["-sdk", pipes.quote(args.sysroot)])
```

Comment:
Yes its using the old arguments form but using array literal did not work because of some commands were separated by a `;`